homeassistant/components/solaredge/coordinator.py

```diff
@@ -216,6 +216,11 @@ def update(self) -> None:
             if len(meter["values"][0]) == 2:
                 self.data[meter["type"]] = meter["values"][0]["value"]
                 self.attributes[meter["type"]] = {"date": meter["values"][0]["date"]}
+                reset = datetime.strptime(
+                    meter["values"][0]["date"], "%Y-%m-%d %H:%M:%S"
+                )
+                reset = reset.replace(tzinfo=timezone.utc)
```

Comment:
So in which timezone is the source?

Reply:
I recheck API, but time should be localize from SolarEdge with the solar panels' location